buffer/src/main/java/io/netty/buffer/ByteBuf.java

```diff
@@ -280,6 +295,7 @@
      * @deprecated use the Little Endian accessors, e.g. {@code getShortLE}, {@code getIntLE}
      * instead of creating a buffer with swapped {@code endianness}.
      */
+    @Deprecated
```

Comment:
Just fixing small broken windows - I'd insist that we should fix small inspection warnings whenever possible unless they are too many.